### PR TITLE
Update E_STOPWATCH FB

### PR DESCRIPTION
**Pull request by franz-hoepfinger-4diac**
Update E_STOPWATCH FB with additional events and event handling logic. Adjusted event input names, output connections, and state transitions for new functionalities. Added algorithms for triggering and resetting the stopwatch.

### CODE REVIEW

**Comment by franz-hoepfinger-4diac** [comment]
https://github.com/eclipse-4diac/4diac-ide/pull/561

**Comment by franz-hoepfinger-4diac** [comment]
https://github.com/eclipse-4diac/4diac-ide/pull/427

File: src/modules/utils/E_STOPWATCH_fbt.h [hunk 1]
```diff
@@ -1,7 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018 fortiss GmbH
- *               2020 Johannes Kepler University Linz
- *               2024 HR Agrartechnik GmbH
+ * Copyright (c) 2018, 2024 fortiss GmbH, Johannes Kepler University Linz, HR Agrartechnik GmbH
```
Comment (franz-hoepfinger-4diac) [inline src/modules/utils/E_STOPWATCH_fbt.h:2]: @azoitl here as well

File: src/modules/utils/E_STOPWATCH_fbt.cpp [hunk 2]
```diff
@@ -1,7 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018 fortiss GmbH
- *               2020 Johannes Kepler University Linz
- *               2024 HR Agrartechnik GmbH
+ * Copyright (c) 2018, 2024 fortiss GmbH, Johannes Kepler University Linz, HR Agrartechnik GmbH
```
Comment (franz-hoepfinger-4diac) [inline src/modules/utils/E_STOPWATCH_fbt.cpp:2]: @azoitl here as well

**Review by azoitl — APPROVED** [review]

**Pull request merged**